pkg/comp-functions/functions/miniodev/objectbucket.go

```diff
@@ -0,0 +1,76 @@
+package miniodev
+
+import (
+	"context"
+
+	crossplanev1 "github.com/crossplane/crossplane/apis/apiextensions/fn/io/v1alpha1"
+	appcatv1 "github.com/vshn/appcat/apis/v1"
+	"github.com/vshn/appcat/pkg/comp-functions/runtime"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ProvisionMiniobucket will create a bucket in the
+func ProvisionMiniobucket(ctx context.Context, iof *runtime.Runtime) runtime.Result {
+	claim := &appcatv1.ObjectBucket{}
```

Comment:
Maybe make it a bit more clear that this is a function only used on dev instance. Also please finish the docstring :sweat_smile: